robotcontainer.py

```diff
@@ -287,6 +288,10 @@ def configureButtonBindings(self):
             ShootBall(self.indexer)
         ).whenReleased(HoldBall(self.indexer))
 
+        commands2.button.JoystickButton(
+            *self.operatorInterface.stopMovingParts
+        ).toggleWhenPressed(StopMovingParts(self.indexer, self.shooter))
```

Comment:
Recommend against using toggle:
From https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html :
>toggleWhenActive/toggleWhenPressed:
This binding toggles a command, scheduling it when a trigger changes from inactive to active (or a button is initially pressed), and canceling it under the same condition if the command is currently running. Note that while this functionality is supported, **toggles are not a highly-recommended option for user control**, as they require the driver to keep track of the robot state. The preferred method is to use two buttons; one to turn on and another to turn off.

Should be one button to enter end-game mode and one to go back to teleop-mode (in case end-game mode was accidentally activated)